lib/utils/utils.dart

```diff
@@ -21,3 +26,20 @@ class _TimestampFromStringJsonConverter
   String toJson(DateTime object) =>
       dateSecondsSinceEpochJsonConverter.toJson(object).toString();
 }
+
+class NtpTime {
+  NtpTime._();
+
+  /// Returns current time of the system clock with offset from the NTP server
+  /// Use this method instead of [DateTime.now] to get the correct time
+  static DateTime now() => inject<NtpService>().now();
+
+  static DateTime clockNow() =>
+      // ignore: avoid_datetime_now
+      Platform.environment.containsKey('FLUTTER_TEST') ? clock.now() : now();
+
+  /// Updates offset from the NTP server
+  /// Use this method to update the offset if you need to get the correct time
+  /// before sensitive operations
+  static void update() => inject<NtpService>().update();
```

Comment:
This is a future-based operation, so leave this method async too

Reply:
🤟